[2024-05-24T08:33:31.172+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-24T08:33:31.198+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task scheduled__2024-05-23T00:00:00+00:00 [queued]>
[2024-05-24T08:33:31.203+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task scheduled__2024-05-23T00:00:00+00:00 [queued]>
[2024-05-24T08:33:31.206+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-24T08:33:31.219+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): first_task> on 2024-05-23 00:00:00+00:00
[2024-05-24T08:33:31.225+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1327) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-24T08:33:31.224+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'first_task', 'scheduled__2024-05-23T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmplx6ljqwz']
[2024-05-24T08:33:31.227+0000] {standard_task_runner.py:91} INFO - Job 30: Subtask first_task
[2024-05-24T08:33:31.226+0000] {standard_task_runner.py:63} INFO - Started process 1338 to run task
[2024-05-24T08:33:31.257+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v2.first_task scheduled__2024-05-23T00:00:00+00:00 [running]> on host 6bbd6353fbe6
[2024-05-24T08:33:31.309+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gauti' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-23T00:00:00+00:00'
[2024-05-24T08:33:31.310+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-24T08:33:31.317+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-24T08:33:31.317+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello everyone, this is our first task!!!']
[2024-05-24T08:33:31.321+0000] {subprocess.py:86} INFO - Output:
[2024-05-24T08:33:31.322+0000] {subprocess.py:93} INFO - hello everyone, this is our first task!!!
[2024-05-24T08:33:31.322+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-24T08:33:31.322+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-24T08:33:31.332+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=first_task, run_id=scheduled__2024-05-23T00:00:00+00:00, execution_date=20240523T000000, start_date=20240524T083331, end_date=20240524T083331
[2024-05-24T08:33:31.369+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-24T08:33:31.376+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-24T08:33:31.376+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
